packages/blockchain-wallet-v4/src/redux/data/sfox/reducers.js

```diff
@@ -60,69 +60,69 @@ const sfoxReducer = (state = INITIAL_STATE, action) => {
     case AT.FETCH_SFOX_ACCOUNTS_FAILURE: {
```

Comment:
Please delete instead of commenting out, why is `FETCH_SFOX_ACCOUNTS_FAILURE ` still around?